addons/udes_stock/models/stock_picking.py

```diff
@@ -627,6 +627,38 @@ def _create_batch(self, pickings):
         PickingBatch = self.env["stock.picking.batch"]
         PickingBatch.create({"picking_ids": [(6, 0, pickings.ids)]})
 
+    def _prepare_individual_move_values(self, picking, product_info, **kwargs):
```

Comment:
Nit: Self can be picking i think
```suggestion
    def _prepare_individual_move_values(self, product_info, **kwargs):
```

Reply:
Oops, yeah it can